build_system/seqan3-config-version.cmake

```diff
@@ -0,0 +1,48 @@
+# -----------------------------------------------------------------------------------------------------
+# Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+# Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+# This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+# shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+# -----------------------------------------------------------------------------------------------------
+
+# This file adds version support for `find_package(SEQAN3 3.1)`.
+
```

Comment:
can you please document that this was auto generated by which script in which version and a link to it?